docs/admin/limitrange/README.md

```diff
@@ -86,30 +86,39 @@ Let's create a simple limit in our namespace.
 
 ```console
 $ kubectl create -f docs/admin/limitrange/limits.yaml --namespace=limit-example
-limitranges/mylimits
+limitrange "mylimits" created
 ```
 
 Let's describe the limits that we have imposed in our namespace.
 
 ```console
 $ kubectl describe limits mylimits --namespace=limit-example
-Name:   mylimits
-Type      Resource  Min  Max Default
-----      --------  ---  --- ---
-Pod       memory    6Mi  1Gi -
-Pod       cpu       250m   2 -
-Container memory    6Mi  1Gi 100Mi
-Container cpu       250m   2 250m
+Name:		mylimits
+Namespace:	limit-example
+Type		Resource	Min	Max	Request	Limit	Limit/Request
+----		--------	---	---	-------	-----	-------------
+Pod		cpu		200m	2	500m	1	-
```

Comment:
request and limit are not supported for type Pod, just Container.

I will make a note to do a validation error if we encounter this.